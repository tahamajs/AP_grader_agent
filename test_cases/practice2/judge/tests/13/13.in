reserve_table 9 1
reserve_table 10 1
reserve_table 3 2
switch 9
switch 9
switch 9
show_table_info 1
switch 10
show_table_info 2
switch 10
show_table_info 2
switch 10
switch 3
switch 9
show_table_info 2
show_table_info 1
