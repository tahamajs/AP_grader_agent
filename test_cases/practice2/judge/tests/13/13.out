Sadegh sits at table 1
Hamid sits at table 1
Parsa sits at table 2
Sadegh switches seats with Hamid!
Sadegh switches seats with Hamid!
Sadegh switches seats with Hamid!
Table ID: 1
People at the table: Hamid, Sadegh
Table remaining capacity: 2
Waiting queue length: 0
Hamid switches seats with Parsa!
Table ID: 2
People at the table: Hamid
Table remaining capacity: 3
Waiting queue length: 0
Hamid switches seats with Parsa!
Table ID: 2
People at the table: Parsa
Table remaining capacity: 3
Waiting queue length: 0
Hamid switches seats with Parsa!
Parsa switches seats with Sadegh!
Sadegh switches seats with Hamid!
Table ID: 2
People at the table: Sadegh
Table remaining capacity: 3
Waiting queue length: 0
Table ID: 1
People at the table: Hamid, Parsa
Table remaining capacity: 2
Waiting queue length: 0
